lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -93,6 +93,34 @@ function M.select_compiler_args(repo, compiler)
   end
 end
 
+function M.select_compile_command(repo, cc, compile_location)
+  if string.match(cc, "cl$") or string.match(cc, "cl.exe$") or not repo.use_makefile then
```

Comment:
@theHamsta This is why it is failing :) `cc` is `clang` so it will go to `make` branch it does not find it, but probably the sequence with `pushd` and `popd` returns 0

Reply:
```suggestion
  if string.match(cc, "cl$") or string.match(cc, "cl.exe$") or not repo.use_makefile or fn.has "win32" == 1 then
```